README.md

```diff
@@ -414,7 +414,33 @@ impl Enum {
 }
 
 assert_eq!(Enum::A(A).get_toto(), <A as WithConst>::TOTO);
+```
+
+- Custom called expression
+
+The `#[expr()]` attribute is used to modify the delegate call, using `$` as a placeholder for what delegate would normaly expand to.
```

Comment:
```suggestion
The `#[expr()]` attribute can be used to modify the delegated call. You can use the `$` sigil as a placeholder for what delegate would normally expand to, and wrap that expression with custom code.
```

Let's also move this to the top, say after `Use modifiers that alter the generated method body`.